sync/downstream.py

```diff
@@ -277,14 +269,7 @@ def next_try_push(self, try_cls=trypush.TryFuzzyCommit):
         self.affected_tests()
 
         action = self.next_action
-        if action == DownstreamAction.try_push:
```

Comment:
I think we can keep this? Something like an infra change doesn't require a stability try push, but we should push to try to ensure the world didn't break